packages/models/src/lib/plugin-config-runner.ts

```diff
@@ -1,13 +1,21 @@
 import { z } from 'zod';
 import { filePathSchema } from './implementation/schemas';
 
+export const outputTransformSchema = z
+  .function()
+  .args(z.array(z.record(z.string(), z.unknown())));
+// @TODO add returns definition as AuditOutputs - this fails ATM when compiling
```

Comment:
Here I could need some help

Reply:
Done, pushed a fix. The problem was a circular dependency caused by `pluginOutputSchema`, but that was unused so I just removed it.

Reply:
how did u figure out the dept?

Reply:
I got there through the `TypeError: Cannot read properties of undefined (reading 'merge')` error. This often means a circular dependency error, it seems (Zod couldn't create the object schema).